src/styles/components/_home.scss

```diff
@@ -39,6 +39,10 @@ $breakpoint-top-section-scale-up: 70rem;
     }
   }
 
+  // &-top__image-column{
+  //   flex-basis: 40%;
```

Comment:
If you don't need these comments, let's delete them!

Reply:
Done!